code/modules/mob/death.dm

```diff
@@ -10,5 +10,5 @@
 	return
 
 /mob/proc/death(gibbed)
-	SEND_SIGNAL(src, COMSIG_GLOB_MOB_DEATH, gibbed)
-	SEND_GLOBAL_SIGNAL(COMSIG_GLOB_MOB_DEATH, src , gibbed)
```

Comment:
remove this, it makes no sense for a mob to die when you can only die if you are living in the first place??